### PR TITLE
Add helper scripts for deployments on osmosis and instructions

### PR DESCRIPTION
**Pull request by lubkoll**

### CODE REVIEW

File: scripts/generate_signed_upload_tx.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+WASM_FILE=$1
+DEPLOYER=$2
+
+FEES=1000000uosmo
+NODE=https://osmosis-rpc.publicnode.com:443
```
Comment (magiodev) [inline scripts/generate_signed_upload_tx.sh:7]: If we hardcode it maybe let's use the official Osmosis RPC: `https://rpc.osmosis.zone:443` ?

Comment (lubkoll) [reply scripts/generate_signed_upload_tx.sh]: done

File: scripts/generate_signed_upload_tx.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+WASM_FILE=$1
+DEPLOYER=$2
+
```
[the remark this answers is not in the document]
Comment (lubkoll) [reply scripts/generate_signed_upload_tx.sh:14]: done

File: scripts/generate_signed_upload_tx.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+FEES=1000000uosmo
+NODE=https://osmosis-rpc.publicnode.com:443
+MULTISIG=osmo1vxq5h3encfyguulqeh26l8dlw9lavl3e2zw7n8
+CHAIN=osmosis-1
+
+set -e
+
+osmosisd tx wasm store ${WASM_FILE} --from contract-upload --gas 25000000 --fees ${FEES} --chain-id ${CHAIN} --node ${NODE} --generate-only > tx.json
```
[the remark this answers is not in the document]
Comment (lubkoll) [reply scripts/generate_signed_upload_tx.sh]: done

**Review by LaurensKubat — APPROVED** [review]
lgtm

**Pull request merged**